crates/sui-pg-temp-db/src/lib.rs

```diff
@@ -315,29 +315,3 @@ fn get_ephemeral_port() -> std::io::Result<u16> {
 
     Ok(addr.port())
 }
-
```

Comment:
Can we not keep this test here, rather than introduce it in `sui_indexer_alt`?

Reply:
technically we can by having a separate `Connection` here, while I think it's useful to test TempDB with `Connection` of indexer and indexer-alt crates respectively -- so `temp_db_smoketest` is now more of a test of integrating TempDB to indexer / indexer-alt.